### PR TITLE
chore(repos): update autoware.repos for autoware_common

### PR DESCRIPTION
**Pull request by yuasabe**
## Description

Update autoware.repos for autoware_common version to point to awsim-stable branch.
Related to: https://github.com/autowarefoundation/autoware/pull/3539

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by taikitanaka3 — APPROVED** [review]
LGTM

File: .pre-commit-config-ansible.yaml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 repos:
   - repo: https://github.com/ansible/ansible-lint.git
-    rev: v6.9.0
+    rev: v6.16.1
```
Comment (taikitanaka3) [inline .pre-commit-config-ansible.yaml:3]: @yuasabe 
この変更でansible通りそうですかね？（そんなことはなかったけど、requiredではないから良いか）

Comment (yuasabe) [reply .pre-commit-config-ansible.yaml:3]: @taikitanaka3 
通ってはいませんが、エラーは変わったみたいです．
https://github.com/AutomotiveAIChallenge/autoware/actions/runs/5139905593/jobs/9250840623?pr=4

ansible-lintの理解があまりまだないのですが、これはちゃんとlintできてる感じでしょうか．

この修正の意図としては、以下の通りです．
- https://github.com/ansible/ansible-lint/issues/3408#issuecomment-1539690205
- [awf/autowareのmainブランチではansible-lintは6.16系](https://github.com/autowarefoundation/autoware/commit/391a85b8a7515e7de0f3946a56894d48705c683b)になっている

Comment (taikitanaka3) [reply .pre-commit-config-ansible.yaml:3]: @yuasabe 
これも必要ですかね？
https://github.com/autowarefoundation/autoware/pull/3490

Comment (yuasabe) [reply .pre-commit-config-ansible.yaml:3]: @taikitanaka3 はい、そう思います．
あと追加で、https://github.com/autowarefoundation/autoware/pull/3500 も入れておいたほうが良い気がします．
こちら自分の方でやっておきます！

**Pull request merged**